docs/notes/2.22.x.md

```diff
@@ -24,6 +24,12 @@ If you encounter such discrepancies, and you can't resolve them easily, please [
 
 ### Backends
 
+#### BUILD
+
+Keys in `overrides` will now be removed from the target parameters. This fix a bug where a parametrized default would have inconsistent parameters with its values.
+
+`parametrize` with single value are now handled the same as not using `parametrized`. No parameter will be added to the target.
```

Comment:
I think this is hard to explain with words, and an example would go miles to complement these paragraphs, if you don't mind adding one. (perhaps just a link to the fixed issue is enough to keep the notes brief?)

Reply:
Are you talking just about the second notes or the first as well?

Reply:
yea, the first one (while the second one being up for discussion how we want to treat it)